### ISSUE TITLE
module resolution: baseURL and paths compiler options don't work in .spec.ts files

### ISSUE THREAD

**Issue by marcelh-gh** [palantir/tslint#3609]
### Bug Report

- __TSLint version__: 5.8.0
- __TypeScript version__: 2.6.2
- __Running TSLint via__:VSCode 

with `tslint.json` configuration:

```json
        "baseUrl": "./src",
        "paths": {
            "*": [
                "A/*",
                "B/*"
            ]  
        }  
```
Using the `baseUrl `and `paths `compiler options works fine for imports in .ts files.
But for import statements in .spec.ts files false error messages are generated.

**Comment by JoshuaKGoldberg** [comment]
Closing this issue for housekeeping; @marcelh-gh if you have more repro information please do post here and we can take a look!
